### PR TITLE
Add pie recipe; fixes #11

### LINKED ISSUE #11: Not enough pastry
There needs to be more pastry recipes

### PR DESCRIPTION
**Pull request by emilieskytte**

### CODE REVIEW

**Comment by rasmusongit** [comment]
i think the recipe is bad, see recommendation

File: pastry/pie.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# pie crust
+
+## Ingredients
+- **100 g** wheat flour
```
Comment (rasmusongit) [inline pastry/pie.md:4]: Put sugar in here somewhere

File: pastry/pie.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# pie crust
+
+## Ingredients
+- **100 g** wheat flour
+- **100 g** rye flour
+- **0.5 dL** olive oil
+- **1 dL** water
+
```
Comment (emilieskytte) [inline pastry/pie.md:8]:
```suggestion

- sugar
```

Comment (rasmusongit) [inline pastry/pie.md:8]:
```suggestion

- **1 kg** sugar
```

File: pastry/pie.md (hunk 1 above)
Comment (emilieskytte) [reply pastry/pie.md:4]: Thank you for the suggestion

File: pastry/pie.md (hunk 2 above)
Comment (rasmusongit) [reply pastry/pie.md:8]: Not very accurate I think

**Pull request merged**